lcls-twincat-common-components/Library/Devices/PPM/FB_PPM_PowerMeter.TcPOU

```diff
@@ -288,24 +288,26 @@ fPulseWattage := SEL(fResponsivity <> 0, 0, (fVoltage - fBackgroundVoltage) / fR
 fCalibMJ := SEL(fFrequency <> 0, -9999, fPulseWattage / fFrequency);
 
 // FF in case voltage is high enough to damage power meter based on wavelength
-{IF defined (K)}
-    CASE REAL_TO_UINT(PMPS_GVL.stCurrentBeamParameters.neV) OF
-        0 .. uSoftWavelengthEdge1L: bOverAllowableEnergy := fCalibMJ > 2;
-        uSoftWavelengthEdge1 .. uSoftWavelengthEdge2L : bOverAllowableEnergy := fCalibMJ > 2;
-        uSoftWavelengthEdge2 .. uSoftWavelengthEdge3L: bOverAllowableEnergy := fCalibMJ > 4;
-        uSoftWavelengthEdge3 .. uSoftWavelengthEdge4L: bOverAllowableEnergy := fCalibMJ > 0.5;
-        uSoftWavelengthEdge4 .. 65535: bOverAllowableEnergy := fCalibMJ > 1;
-    END_CASE
-{ELSIF defined (L)}
-    CASE REAL_TO_UINT(PMPS_GVL.stCurrentBeamParameters.neV) OF
-        uHardWavelengthEdge1 .. uHardWavelengthEdge2L: bOverAllowableEnergy := fCalibMJ > 10;
-        uHardWavelengthEdge2 .. uHardWavelengthEdge3L: bOverAllowableEnergy := fCalibMJ > 4;
-        uHardWavelengthEdge3 .. 65535: bOverAllowableEnergy := fCalibMJ > 10;
-    END_CASE
-{END_IF}
-FFO(i_xOK := NOT bOverAllowableEnergy,
-    i_DevName := sDeviceName,
-    io_fbFFHWO := fbFFHWO);
+IF rTrig_Background.Q THEN
+    {IF defined (K)}
+        CASE REAL_TO_UINT(PMPS_GVL.stCurrentBeamParameters.neV) OF
+            0 .. uSoftWavelengthEdge1L: bOverAllowableEnergy := fCalibMJ > 2;
+            uSoftWavelengthEdge1 .. uSoftWavelengthEdge2L : bOverAllowableEnergy := fCalibMJ > 2;
+            uSoftWavelengthEdge2 .. uSoftWavelengthEdge3L: bOverAllowableEnergy := fCalibMJ > 4;
+            uSoftWavelengthEdge3 .. uSoftWavelengthEdge4L: bOverAllowableEnergy := fCalibMJ > 0.5;
+            uSoftWavelengthEdge4 .. 65535: bOverAllowableEnergy := fCalibMJ > 1;
+        END_CASE
+    {ELSIF defined (L)}
+        CASE REAL_TO_UINT(PMPS_GVL.stCurrentBeamParameters.neV) OF
+            uHardWavelengthEdge1 .. uHardWavelengthEdge2L: bOverAllowableEnergy := fCalibMJ > 10;
+            uHardWavelengthEdge2 .. uHardWavelengthEdge3L: bOverAllowableEnergy := fCalibMJ > 4;
+            uHardWavelengthEdge3 .. 65535: bOverAllowableEnergy := fCalibMJ > 10;
+        END_CASE
+    {END_IF}
+    FFO(i_xOK := NOT bOverAllowableEnergy,
```

Comment:
Super late for me to be catching this, but calling this `FFO` is confusing because it's not a fast fault output, it's just a fast fault. Probably too nitpicky, sorry